src/section.js

```diff
@@ -81,23 +81,14 @@ class Section {
 	 * @param  {method} _request a request method to use for loading
 	 * @return {string} output a serialized XML Document
 	 */
-	render(_request){
+	render(){
 		var rendering = new defer();
 		var rendered = rendering.promise;
 		this.output; // TODO: better way to return this from hooks?
 
-		this.load(_request).
+		this.load().
 			then(function(contents){
-				var serializer;
```

Comment:
@fchasen We have added the serializer, but we are using **xmldom** serializer.
We are using **xmldom** becasue in IE we have a issue where the default browser serializer genrates the html which is having self closing tags(Ex. script tags and other tags), which IE does't understands and does't render it properly.